lib/optimizely/decision_service.rb

```diff
@@ -239,50 +220,46 @@ def get_variation_for_feature_rollout(feature_flag, user_id, attributes = nil)
 
```

Comment:
Replace "Feature flag '#{feature_flag_key}' is not part of a rollout." by
"Feature flag '#{feature_flag_key}' is not used in a rollout."

Reply:
I don't see this update

Reply:
Done.